lib/exgboost/parameters.ex

```diff
@@ -0,0 +1,968 @@
+defmodule EXGBoost.Parameters do
+  @global_params [
+    verbosity: [
+      type: {:custom, EXGBoost.Parameters, :validate_verbosity, []},
+      doc: """
+      Verbosity of printing messages. Valid values are:
+      `:silent`, `:warning`, `:info`, `:debug`.
+      """
+    ],
+    use_rmm: [
+      type: :boolean,
+      default: false,
+      doc: """
+      Whether to use RAPIDS Memory Manager for memory allocation.
+      This This option is only applicable when XGBoost is built (compiled)
+      with the RMM plugin enabled. Valid values are `true` and `false`.
+      """
+    ]
+  ]
+
+  @general_params [
+    booster: [
+      type: {:in, [:gbtree, :gblinear, :dart]},
+      default: :gbtree,
+      doc: """
+      Which booster to use. Valid values are `:gbtree`, `:gblinear`, `:dart`
+          * `:gbtree`: tree-based models
+          * `:gblinear`: linear models
+          * `:dart`: tree-based models with dropouts
```

Comment:
```suggestion
          * `:gbtree` - tree-based models
          * `:gblinear` - linear models
          * `:dart` - tree-based models with dropouts
```

I think this looks better